src/plots/cartesian/tick_value_defaults.js

```diff
@@ -22,22 +22,53 @@ module.exports = function handleTickValueDefaults(containerIn, containerOut, coe
     }
 
     if(Array.isArray(containerIn.tickvals)) tickmodeDefault = 'array';
-    else if(containerIn.dtick && isNumeric(containerIn.dtick)) {
```

Comment:
... and probably one of the most useful new features of the passed few months :trophy: